source/js/buyers-guide/bg-main.js

```diff
@@ -1,4 +1,3 @@
-import React from "react";
```

Comment:
https://reactjs.org/blog/2020/09/22/introducing-the-new-jsx-transform.html#removing-unused-react-imports